### PR TITLE
FRW-1610 Adjusted docs

### PR DESCRIPTION
**Pull request by dimitriyTsemma**
Ticket: https://spryker.atlassian.net/browse/FRW-1613

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 1]
```diff
@@ -86,17 +82,17 @@ class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiA
     protected function getApplicationPlugins(): array
     {
         return [
-            new StoreHttpHeaderApplicationPlugin(),
+            new StoresApplicationPlugin(),
```
Comment (asmarovydlo) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:85]: Plugin name change requested in PR, please rename it here accordingly.

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 2]
```diff
@@ -132,7 +139,7 @@ class GlueStorefrontApiApplicationDependencyProvider extends SprykerGlueStorefro
     {
         return [
             new HttpApplicationPlugin(),
-            new StoreHttpHeaderApplicationPlugin(),
+            new StoresApplicationPlugin(),
```
Comment (asmarovydlo) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:142]: Plugin name change requested in PR, please rename it here accordingly.

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 3]
```diff
@@ -14,10 +14,12 @@ To start feature integration, overview and install the necessary features:
 
 Activate the following plugins:
 
-| PLUGIN | SPECIFICATION | PREREQUISITES | NAMESPACE |
-| --- | --- | --- | --- |
-| StoreHttpHeaderApplicationPlugin | Gets store name from the Request header. | None | Spryker\Glue\StoresRestApi\Plugin\Application |
-| LocaleApplicationPlugin | Gets locale name from the Request header. | None | Spryker\Glue\ProductOptionsRestApi\Plugin\GlueApplication |
+| PLUGIN | SPECIFICATION                                                                                                                                  | PREREQUISITES | NAMESPACE                                                 |
+| --- |------------------------------------------------------------------------------------------------------------------------------------------------| --- |-----------------------------------------------------------|
+| StoreHttpHeaderApplicationPlugin | Gets store name from the Request parameter or Request header used for Glue Application. If both defined Request parameter has more priority. - | None | Spryker\Glue\StoresRestApi\Plugin\Application             |
```
Comment (asmarovydlo) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:19]: Please add a warning that `StoresRestApi` is deprecated, and we need to use `StoresApi` instead.

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 4]
```diff
@@ -75,7 +76,7 @@ Make sure that the store and locale are set correctly.
 namespace Pyz\Glue\GlueBackendApiApplication;
 
 use Spryker\Glue\GlueBackendApiApplication\GlueBackendApiApplicationDependencyProvider as SprykerGlueBackendApiApplicationDependencyProvider;
-use Spryker\Glue\StoresRestApi\Plugin\Application\StoreHttpHeaderApplicationPlugin;
+use Spryker\Glue\StoresBackendApi\Plugin\Application\StoreApplicationPlugin;
```
Comment (asmarovydlo) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:79]: Please fix namespaces, to match the code in the PR after refactoring. (Same for SAPI)

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 5]
```diff
@@ -14,11 +14,18 @@ To start feature integration, overview and install the necessary features:
 
 Activate the following plugins:
 
-| PLUGIN | SPECIFICATION | PREREQUISITES | NAMESPACE |
-| --- | --- | --- | --- |
-| StoreHttpHeaderApplicationPlugin | Gets store name from the Request header. | None | Spryker\Glue\StoresRestApi\Plugin\Application |
-| LocaleApplicationPlugin | Gets locale name from the Request header. | None | Spryker\Glue\ProductOptionsRestApi\Plugin\GlueApplication |
+| PLUGIN | SPECIFICATION                                                                                                                                  | PREREQUISITES | NAMESPACE                                                 |
+| --- |------------------------------------------------------------------------------------------------------------------------------------------------| --- |-----------------------------------------------------------|
+| StoreHttpHeaderApplicationPlugin | Gets store name from the Request parameter or Request header used for Glue Application. If both defined Request parameter has more priority. - | None | Spryker\Glue\StoresRestApi\Plugin\Application             |
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:19]:
```suggestion
| StoreHttpHeaderApplicationPlugin | Gets the name of the store from the Request parameter or the Request header used for the Glue Application. If both are defined, the Request parameter has more priority. - | None | Spryker\Glue\StoresRestApi\Plugin\Application             |
```

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 6]
```diff
@@ -14,11 +14,18 @@ To start feature integration, overview and install the necessary features:
...
+| StoreApplicationPlugin | Gets store name from the Request parameter or Request header used for Storefront API. If both defined Request parameter has more priority. -   | None | Spryker\Glue\StoresApi\Plugin\GlueStorefrontApiApplication                 |
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:20]:
```suggestion
| StoreApplicationPlugin | Gets the name of the store from the Request parameter or the Request header used for the Storefront API. If both are defined, the Request parameter has more priority. -   | None | Spryker\Glue\StoresApi\Plugin\GlueStorefrontApiApplication                 |
```

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 7]
```diff
@@ -14,11 +14,18 @@ To start feature integration, overview and install the necessary features:
...
+| StoreApplicationPlugin | Gets store name from the Request parameter or Request header used for Storefront API. If both defined Request parameter has more priority. -   | None | Spryker\Glue\StoresBackendApi\Plugin\GlueBackendApiApplication          |
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:21]:
```suggestion
| StoreApplicationPlugin |  Gets the name of the store from the Request parameter or the Request header used for the Storefront API. If both are defined, the Request parameter has more priority. -   | None | Spryker\Glue\StoresBackendApi\Plugin\GlueBackendApiApplication          |
```

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 8]
```diff
@@ -50,18 +57,12 @@ class GlueApplicationDependencyProvider extends SprykerGlueApplicationDependency
 
 {% info_block warningBox "Verification" %}
 
-If everything is set up correctly, a request to `https://glue.mysprykershop.com` with the header `[{"key":"Accept-Language","value":"de_DE"},{"key":"Store","value":"DE"}]` should result in a response whithout any errors and contains the `content-language` header set to **de_DE**.
+If everything is set up correctly, a request to `https://glue.mysprykershop.com` with the header `[{"key":"Accept-Language","value":"de_DE"},{"key":"Store","value":"DE"}]` or `https://glue.mysprykershop.com?_store=DE`  with the header `[{"key":"Accept-Language","value":"de_DE"}]` should result in a response whithout any errors and contains the `content-language` header set to **de_DE**.
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:60]:
```suggestion
If everything is set up correctly, a request to `https://glue.mysprykershop.com` with the header `[{"key":"Accept-Language","value":"de_DE"},{"key":"Store","value":"DE"}]` or `https://glue.mysprykershop.com?_store=DE`  with the header `[{"key":"Accept-Language","value":"de_DE"}]` should result in a response without any errors and contain the `content-language` header set to **de_DE**.
```

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 9]
```diff
@@ -86,17 +87,17 @@ class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiA
...
+            new StoreApplicationPlugin(),
         ];
     }
 }
 ```
 
 {% info_block warningBox "Verification" %}
 
-If everything is set up correctly, a request to `https://glue-backend.mysprykershop.com` with the header `[{"key":"Store","value":"DE"}]` should result in a response without any errors.
+If everything is set up correctly, a request to `https://glue-backend.mysprykershop.com` with the header `[{"key":"Accept-Language","value":"de_DE"},{"key":"Store","value":"DE"}]` or `https://glue.mysprykershop.com?_store=DE`  with the header `[{"key":"Accept-Language","value":"de_DE"}]` should result in a response whithout any errors and contains the `content-language` header set to **de_DE**.
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:98]:
```suggestion
If everything is set up correctly, a request to `https://glue-backend.mysprykershop.com` with the header `[{"key":"Accept-Language","value":"de_DE"},{"key":"Store","value":"DE"}]` or `https://glue.mysprykershop.com?_store=DE`  with the header `[{"key":"Accept-Language","value":"de_DE"}]` should result in a response without any errors and contain the `content-language` header set to **de_DE**.
```

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 10]
```diff
@@ -86,17 +87,17 @@ class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiA
...
 
-An example for testing: 
+An example for testing with Store header:
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:100]:
```suggestion
An example for testing with a Store header: 
```

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 11]
```diff
@@ -108,6 +109,17 @@ curl --location --request POST 'http://glue-backend.eu.mysprykershop.com/token'
 
 ```
 
+An example for testing with Store Request parameter:
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:112]:
```suggestion
An example for testing with a Store Request parameter:
```

File: _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md [hunk 12]
```diff
@@ -141,6 +153,6 @@ class GlueStorefrontApiApplicationDependencyProvider extends SprykerGlueStorefro
 
 {% info_block warningBox "Verification" %}
 
-If everything is set up correctly, a request to `https://glue-storefront.mysprykershop.com` with the header `[{"key":"accept-language","value":"de_DE"},{"key":"Store","value":"DE"}]` should result in a response without any errors and will contain the `content-language` header set to **de_DE**.
+If everything is set up correctly, a request to `https://glue-storefront.mysprykershop.com` with the header `[{"key":"Accept-Language","value":"de_DE"},{"key":"Store","value":"DE"}]` or `https://glue.mysprykershop.com?_store=DE`  with the header `[{"key":"Accept-Language","value":"de_DE"}]` should result in a response whithout any errors and contains the `content-language` header set to **de_DE**.
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202304.0/install-glue-api/install-the-dynamic-store-glue-api.md:156]:
```suggestion
If everything is set up correctly, a request to `https://glue-storefront.mysprykershop.com` with the header `[{"key":"Accept-Language","value":"de_DE"},{"key":"Store","value":"DE"}]` or `https://glue.mysprykershop.com?_store=DE`  with the header `[{"key":"Accept-Language","value":"de_DE"}]` should result in a response without any errors and contain the `content-language` header set to **de_DE**.
```

**Pull request merged**